Templates/Integrations/Authentication/Authentication.py

```diff
@@ -0,0 +1,410 @@
+        """Returns events by the account ID.
+
+        Args:
+            account_id: Account ID to unlock.
+
+        Returns:
+            Response JSON
+        """
+        # The service endpoint to request from
+        suffix = 'account/unlock'
+        # Dictionary of params for the request
+        params = {'account': account_id}
+        # Send a request using our http_request wrapper
+        return self._http_request('POST', suffix, params=params)
+
+    def reset_account_request(self, account_id: str):
+        """Resets an account by account ID.
+
+        Args:
+            account_id: Account ID to reset.
+
+        Returns:
+            Response JSON
+        """
+        # The service endpoint to request from
+        suffix = 'account/reset'
+        # Dictionary of params for the request
+        params = {'account': account_id}
+        # Send a request using our http_request wrapper
+        return self._http_request('POST', suffix, params=params)
+
+    def unlock_vault_request(self, vault_to_lock: AnyStr) -> Dict:
+        """Unlocks a vault by vault ID.
+
+        Args:
+            vault_to_lock: Vault ID to lock
+
+        Returns:
+            Response JSON
+        """
+        suffix = 'vault/unlock'
+        params = {'vaultId': vault_to_lock}
+        return self._http_request('POST', suffix, params=params)
+
+    def lock_vault_request(self, vault_to_lock: AnyStr) -> Dict:
+        """Locks vault by vault ID.
+
+        Args:
+            vault_to_lock: Vault ID to lock.
+
+        Returns:
+            Response JSON
+        """
+        suffix = 'vault/lock'
+        params = {'vaultId': vault_to_lock}
+        return self._http_request('POST', suffix, params=params)
+
+    def list_vaults_request(self, max_results: int) -> Dict:
+        """Return all vaults from API.
+
+        Args:
+            max_results: Maximum results to fetch.
+
+        Returns:
+            Response JSON
+        """
+        suffix = 'vault'
+        values_to_ignore = [0]
+        params = assign_params(limit=max_results, values_to_ignore=values_to_ignore)
+        return self._http_request('GET', suffix, params=params)
+
+
+''' HELPER FUNCTIONS '''
+
+
+def build_account_context(credentials: Union[Dict, List]) -> Union[Dict, List]:
+    """Formats the API response to Demisto context.
+
+    Args:
+        credentials: The raw response from the API call. Can be a List or Dict.
+
+    Returns:
+        The formatted Dict or List.
+
+    Examples:
+        >>> build_account_context([{'username': 'user', 'name': 'demisto', 'isLocked': False}])
+        [{'Username': 'user', 'Name': 'demisto', 'IsLocked': False}]
+    """
+    if isinstance(credentials, list):
+        return [build_account_context(credential) for credential in credentials]
+    return {
+        'Username': credentials.get('username'),
+        'Name': credentials.get('name'),
+        'IsLocked': credentials.get('isLocked'),
```

Comment:
remove trailing comma

Reply:
i disagree.
https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas